src/datastore/parser.cpp

```diff
@@ -0,0 +1,166 @@
+        std::cout << "-----------League: " << (*it)[8] << std::endl;
+        std::cout << "------Date opened: " << (*it)[9] << std::endl;
+        std::cout << "--Distance2CtrFld: " << (*it)[10] << std::endl;
+        std::cout << "Ballpark typology: " << (*it)[11] << std::endl;
+        std::cout << "--------Roof Type: " << (*it)[12] << std::endl;
+        std::cout << "NumberOSouvenirs:  " << (*it)[13] << std::endl;
+        std::cout << "Souvenir ID:  " << (*it)[14] << std::endl;
+        std::cout << "Is deleted?:  " << (*it)[15] << std::endl;
+        std::cout << "---Souvenir:  " << (*it)[16] << std::endl;
+        std::cout << "------Price:  " << (*it)[17] << std::endl;
+        std::cout << "Souvenir ID:  " << (*it)[18] << std::endl;
+        std::cout << "Is deleted?:  " << (*it)[19] << std::endl;
+        std::cout << "---Souvenir:  " << (*it)[20] << std::endl;
+        std::cout << "------Price:  " << (*it)[21] << std::endl;
+        std::cout << "Souvenir ID:  " << (*it)[22] << std::endl;
+        std::cout << "Is deleted?:  " << (*it)[23] << std::endl;
+        std::cout << "---Souvenir:  " << (*it)[24] << std::endl;
+        std::cout << "------Price:  " << (*it)[25] << std::endl;
+        std::cout << "Souvenir ID:  " << (*it)[26] << std::endl;
+        std::cout << "Is deleted?:  " << (*it)[27] << std::endl;
+        std::cout << "---Souvenir:  " << (*it)[28] << std::endl;
+        std::cout << "------Price:  " << (*it)[29] << std::endl;
+        std::cout << "Souvenir ID:  " << (*it)[30] << std::endl;
+        std::cout << "Is deleted?:  " << (*it)[31] << std::endl;
+        std::cout << "---Souvenir:  " << (*it)[32] << std::endl;
+        std::cout << "------Price:  " << (*it)[33] << std::endl;
+        std::cout << std::endl;
+    }
+}
+
+void distanceParseDebug(std::string path)
+{
+    std::vector<std::vector<std::string>> distanceData = loadDistanceData(path);
+    for (std::vector<std::vector<std::string>>::iterator it = distanceData.begin(); it != distanceData.end(); ++it)
+    {
+
+        std::cout << "Stadium ID:   " << (*it)[0] << std::endl;
+        std::cout << "Stadium Name: " << (*it)[1] << std::endl;
+        std::cout << "Stadium ID:   " << (*it)[2] << std::endl;
+        std::cout << "Stadium Name: " << (*it)[3] << std::endl;
+        std::cout << "Distance:     " << (*it)[4] << std::endl;
+        std::cout << std::endl;
+    }
+}
+
+std::vector<std::vector<std::string>> loadTeamData(std::string path)
+{
+    // Initializing - vector will hold all lines of data within the document
+    std::vector<std::vector<std::string>> allRows;
+    std::ifstream infile(path, std::ios::in);
+    int lineCount = 0;
+
+    // Checks if file is open if now throw error
+    if (infile.is_open())
+    {
+        std::string line;
+        while (infile.good())
+        {
+            std::getline(infile, line);
+            if (!infile.eof())
+            {
+                if (line[0] == '#' || line[0] == ' ')
```

Comment:
What is the second operand of the `or` for? If it's for checking an empty line, why not do 
```cpp
line[0].empty()
```

Reply:
We can reduce this `if` statement to one line
```cpp
if(...) { continue; }
```
as per the coding style.

Reply:
It's checking both if there are spaces or if there are comments